resources/ts/Shared/SecondaryLinkButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Link, InertiaLinkProps } from "@inertiajs/react";
+import { classNames } from "../utils/classNames";
+
+type Props = InertiaLinkProps;
+
+export const SecondaryLinkButton = ({
```

Comment:
Same here I prefer to invert the naming. 
So you only need to read the first part of the file to know that it's about a button...

```suggestion
export const ButtonLinkSecondary = ({
```